lang/java/pom.xml

```diff
@@ -210,9 +211,16 @@
           </configuration>
         </plugin>
         <plugin>
-          <groupId>org.codehaus.mojo</groupId>
+          <groupId>org.javacc.plugin</groupId>
           <artifactId>javacc-maven-plugin</artifactId>
           <version>${javacc-plugin.version}</version>
+          <dependencies>
```

Comment:
The new plugin treats JavaCC as a provided dependency, so we need to add it explicitly.